index.js

```diff
@@ -0,0 +1,165 @@
+"use strict";
+//Get days in year
+const getDaysInYear = (year) => {
+  let date = new Date(year, 2);
+  let numberDay = 0;
+  date.setDate(0);
+  const n = date.getDate();
+  n === 28 ? (numberDay = 365) : (numberDay = 366);
+  document.getElementById("outputCountDay").value = numberDay;
+};
+
+//Get day of a year
+const getDayNumber = (date) => {
+  let newDate = new Date(date);
+  const year = newDate.getFullYear();
+  const month = newDate.getMonth();
+  const day = newDate.getDate();
+  let numberDay = 0;
+  for (let i = 0; i < month; i++) {
+    let newDate = new Date(year, i + 1, 0);
+    const newDay = newDate.getDate();
+    numberDay += newDay;
+  }
+  numberDay += day;
+  document.getElementById("outputNumber").value = numberDay;
+};
+
+//Get fiscal quarters
+const months = [
+  "Jan",
+  "Feb",
+  "Mar",
+  "Apr",
+  "May",
+  "Jun",
+  "Jul",
+  "Aug",
+  "Sep",
+  "Oct",
+  "Nov",
+  "Dec",
+];
+let getQuarters = (date) => {
+  date = new Date(date);
+  const month = date.getMonth();
+  const day = date.getDate();
+  let quarter = "";
+  month >= 0 && month <= 2
+    ? (quarter = "I quarter")
+    : month >= 3 && month <= 5
+    ? (quarter = "II quarter")
+    : month >= 6 && month <= 8
+    ? (quarter = "III quarter")
+    : month >= 9 && month <= 11
+    ? (quarter = "IV quarter")
+    : (quarter = "Error");
```

Comment:
My recommendation is to use ?: in cases where you need a return value if not you can use if statement  

`(quarter =  month >= 0 && month <= 2?'I quarter':month >= 3 && month <= 5?... ` 
or
`if(month >= 0 && month <= 2){
quarter = "I quarter
}else ...`

also you can try to use switch here